tools/hcluster_sg_parser/hcluster_sg_parser.xml

```diff
@@ -0,0 +1,30 @@
+<tool id="hcluster_sg_parser" name="hcluster_sg_parser" version="0.1.1" force_history_refresh="True">
+     <description>Converts hcluster_sg 3-column output into list of ids</description>
+     <command>
+     <![CDATA[
+          perl $__tool_directory__/table_row_to_column.pl
+          $inputFile
+     ]]>
+     </command>
+     <inputs>
+          <param name="inputFile" type="data" format="tabular" label="hcluster output file in 3-column format" help="3-columns format: cluster_id cluster-size     integer-weight cluster-member" />
+     </inputs>
+     <outputs>
+     <data format="txt" name="report">
+          <discover_datasets pattern="__designation_and_ext__" visible="true" />
+     </data>
+
+     </outputs>
+     <tests>
+          <test>
+                <param name="inputFile" ftype="tabular" value="hcluster.tabular" />
```

Comment:
Need to add the output test files here. And to add the `test-data/` directory with the files.